src/utils/SiriusAdapter.cpp

```diff
@@ -140,7 +140,7 @@ class TOPPSiriusAdapter :
     registerStringOption_("isotope", "<choice>", "both", "how to handle isotope pattern data. Use 'score' to use them for ranking or 'filter' if you just want to remove candidates with bad isotope pattern. With 'both' you can use isotopes for filtering and scoring (default). Use 'omit' to ignore isotope pattern.", false);
     setValidStrings_("isotope", ListUtils::create<String>("score,filter,both,omit"));
     registerStringOption_("elements", "<choice>", "CHNOP[5]S", "The allowed elements. Write CHNOPSCl to allow the elements C, H, N, O, P, S and Cl. Add numbers in brackets to restrict the maximal allowed occurrence of these elements: CHNOP[5]S[8]Cl[1]. By default CHNOP[5]S is used.", false);
-    registerIntOption_("tree_timeout", "<num>", 10, "Time out in seconds per fragmenation tree computation (default: 10)", false);
+    registerIntOption_("tree_timeout", "<num>", 10, "Time out in seconds per fragmenation tree computation (default: 10). To disable the tree timout set the value to 0", false);
```

Comment:
IIRC the default value is automatically added to the description.

Reply:
Thanks for pointing that out!